cbrkit/sim/collections.py

```diff
@@ -1,6 +1,6 @@
 from collections.abc import Collection, Sequence, Set
-from typing import Any
-
+from typing import Any, Callable, List
+import heapq
```

Comment:
Should be imported in the function itself to optimize performance: If it is not needed, it is not imported